app/models/user.rb

```diff
@@ -20,6 +20,10 @@ class User < ApplicationRecord
   def three_most_recent_posts
     Post.where(author_id: id).order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- Kudos to you both for seeing it fit to create your `user.admin?` method inside the user model. Great job :clap: